### DISCUSSION TITLE
Single dot (".") query and how I could block it?

### DISCUSSION THREAD

**Discussion by mangkoran** [AdguardTeam/AdGuardHome#7055] [Q&A]
Hi all. I noticed my internet feels slow whenever I open a website for the first time. After I checked AGH log, I found a lot of single dot (".") query coming from my router which I believe slows down AGH. What is this query? And how I could block it?

AGH v0.107.48
OpenWRT w/ kernel 6.6.30
dnsmasq forwards to AGH
wireguard-tools 1.0.20210914-r3

![Screenshot_20240603-210215](https://github.com/AdguardTeam/AdGuardHome/assets/57625126/f8a8817e-7788-4afb-a4d5-58d6a4638fc4)
![Screenshot_20240603-211447](https://github.com/AdguardTeam/AdGuardHome/assets/57625126/ae62e44b-ac65-4ade-bceb-5aaab779fc4e)

**Comment by ainar-g** [comment]
See https://github.com/AdguardTeam/AdGuardHome/issues/6049#issuecomment-1654176843.

**Reply by mangkoran** [reply]
Apologize I deleted my previous comments as I have misunderstandings.

After recollecting my findings, I decided to open #7058 instead
